near-sdk/src/environment/env.rs

```diff
@@ -996,7 +1806,14 @@ pub fn state_exists() -> bool {
 
 /// Price per 1 byte of storage from mainnet genesis config.
 /// TODO: will be using the host function when it will be available.
-
+///
```

Comment:
should mention usage in 
https://github.com/near/near-sdk-rs/tree/master/near-contract-standards/src/fungible_token/storage_impl.rs
https://github.com/near/near-sdk-rs/tree/master/near-contract-standards/src/non_fungible_token/utils.rs
https://github.com/near/near-sdk-rs/tree/master/examples/fungible-token/tests/workspaces.rs